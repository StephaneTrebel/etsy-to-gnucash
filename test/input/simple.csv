Date,Type,Titre,Info,Devise,Montant,"Frais Et Taxes",Net,"Informations Fiscales"
"25 septembre 2022",TVA,"TVA: Etsy Ads",,EUR,--,-€0.17,-€0.17,--
"25 septembre 2022",Marketing,"Etsy Ads","Bill for click-throughs to your shop on Sep 24, 2022",EUR,--,-€1.44,-€1.44,--
