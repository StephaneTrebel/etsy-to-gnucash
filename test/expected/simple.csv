ID,Date,Type,Compte,Description,Crédit,Débit
1,25/09/22,Marketing,Dépenses:Services:Etsy:Etsy Ads,"Bill for click-throughs to your shop on Sep 24, 2022","1,44",
1,25/09/22,Marketing,Actifs:Etsy:Wallet,"Bill for click-throughs to your shop on Sep 24, 2022",,"1,44"
1,25/09/22,TVA,Dépenses:Services:Etsy:Etsy Ads,TVA: Etsy Ads (VAT),"0,17",
1,25/09/22,TVA,Actifs:Etsy:Wallet,TVA: Etsy Ads (VAT),,"0,17"
